tempMonApp/src/tempMon.c

```diff
@@ -405,3 +410,57 @@ cpu_thermal_init(int hwmon_inst, struct temp_state* state)
   state->fd = open(file, O_RDONLY);
   return state->fd < 0 ? -1 : 0;
 }
+
+/******************************************************************************
+ * IOC shell function registration
+ *****************************************************************************/
+
+static void
+hwmonStatusCallFunc(const iocshArgBuf* args)
+{
+  /** Caller wants us to re-index all hwmon types */
```

Comment:
In the spirit of using this code as a teaching resource, could you explain somewhere what re-indexing hwmon types means (maybe in README or another source of documentation)? I myself don't know the answer.